src/Tools/IdeCoreBenchmarks/NavigateToBenchmarks.cs

```diff
@@ -167,17 +153,20 @@ public async Task RunProjectParallelIndexing()
             Console.ReadLine();
         }
 
-        //  [Benchmark]
+        [Benchmark]
         public async Task RunFullParallelIndexing()
         {
             Console.WriteLine("Attach now");
-            Console.ReadLine();
+            Thread.Sleep(10000);
```

Comment:
No need to change anything here, but is this for waiting to attach the debugger?  Could do `Debugger.Launch()` which will block until attached

Reply:
thanks!